src/components/molecules/EventPaymentButton/EventPaymentButton.tsx

```diff
@@ -1,89 +0,0 @@
-import React from "react";
-import "firebase/functions";
-import { Link } from "react-router-dom";
-
-import { VenueEvent } from "types/venues";
-
-import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { isUserAMember } from "utils/isUserAMember";
-import { isEventStartingSoon } from "utils/event";
-import { WithId } from "utils/id";
-import { venueEntranceUrl } from "utils/url";
-import {
-  currentVenueSelectorData,
-  userPurchaseHistorySelector,
-} from "utils/selectors";
-
-import { useUser } from "hooks/useUser";
-import { useSelector } from "hooks/useSelector";
-import useConnectUserPurchaseHistory from "hooks/useConnectUserPurchaseHistory";
-
-import "./EventPaymentButton.scss";
-
-interface PropsType {
-  event: WithId<VenueEvent>;
-  venueId: string;
-  setIsPaymentModalOpen: (value: boolean) => void;
-  selectEvent: () => void;
-  paymentConfirmationPending: boolean;
```

Comment:
Have we made sure to remove any code related to `setIsPaymentModalOpen` / `paymentConfirmationPending` that used to be passed in to this component?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40